src/bindings/html/view.js

```diff
@@ -16,6 +17,7 @@ const observerConfig = {
 export class View {
   constructor(doc) {
     this.doc = doc;
```

Comment:
just a thought as I saw it. Why is View.doc public? Does document.l10n.doc make any sense? Should it be at lest prefixed with '_'?